pkg/apis/dashboard/v0alpha1/doc.go

```diff
@@ -1,4 +1,3 @@
-// +k8s:deepcopy-gen=package
```

Comment:
is there a reason do remove it here, but add it explicitly to individual items

If there is a reason *not* to have it -- can you add that as as comment in this file?

Reply:
Sure, I'll add a comment. TL;DR it doesn't work well with SDK-generated code (we already generate deep copy methods in the SDK).